README.md

```diff
@@ -48,7 +53,17 @@ on any transformations. All QGreenland GeoPackages and GeoTIFFs are projected
 in `EPSG:3413`.
 
 
-# Getting started
+# Level of Support
+
+This repository is fully supported by NSIDC. If you discover any problems or
+bugs, please submit an Issue. If you would like to contribute to this
+repository, you may fork the repository and submit a pull request.
+
+See the [LICENSE](LICENSE) for details on permissions and warranties. Please
```

Comment:
The LICENSE is called out in the "Level of Service". I thought this had been under its own section, maybe we revised the template.

Reply:
Yeah, you may be right. This is language taken directly from the template README.